python/mlc_llm/interface/compiler_flags.py

```diff
@@ -64,7 +64,7 @@ def boolean(value: str) -> bool:
         parser.add_argument("--cudagraph", type=boolean, default=False)
         parser.add_argument("--cutlass", type=boolean, default=False)
         parser.add_argument(
-            "--allreduce-strategy",
+            "--allreduce_strategy",
```

Comment:
The usual command line argument convention prefers dashes. We probably don't need to change this file.

Reply:
This was required because initially, it returned a subprocess.CalledProcessError